etna/search/views.py

```diff
@@ -155,7 +155,9 @@ def paginate_api_result(
         """
         paginator = APIPaginator(total_count, per_page=per_page)
         page = PaginatorPage(result_list, number=self.page_number, paginator=paginator)
-        page_range = paginator.get_elided_page_range(number=self.page_number, on_each_side=1, on_ends=1)
+        page_range = paginator.get_elided_page_range(
+            number=self.page_number, on_each_side=1, on_ends=1
```

Comment:
@TNA-Allan This has been updated to match the logic in https://github.com/nationalarchives/ds-wagtail/pull/1466/files